pkg/config/config.go

```diff
@@ -110,11 +115,58 @@ func (c *Config) InitConfig() {
 	}
 }
 
+func (c *Config) EditConfig() error {
+	var err error
+
+	fmt.Println("Opening config file:", c.ProfilesFile)
+
+	switch runtime.GOOS {
+	case "darwin", "linux":
+		cmd := exec.Command(os.Getenv("EDITOR"), c.ProfilesFile)
+		// Some editors detect whether they have control of stdin/out and will
+		// fail if they do not.
+		cmd.Stdin = os.Stdin
+		cmd.Stdout = os.Stdout
+		return cmd.Run()
+	case "windows":
+		// As far as I can tell, Windows doesn't have an easily accesible or
+		// comparable option to $EDITOR, so default to notepad for now
+		err = exec.Command("notepad", c.ProfilesFile).Run()
```

Comment:
This might be fixed in Windows 10, but in the past notepad did not handle LF line endings (only CR+LF Windows-style line endings). We should test to check if it displays the config file correctly.